.github/workflows/release_next.yml

```diff
@@ -5,27 +5,56 @@ on:
       - master
 
 jobs:
-  release:
+  build_artifact:
     if: github.repository == 'hashicorp/terraform-cdk'
     runs-on: ubuntu-latest
     container:
       image: hashicorp/jsii-terraform
     env:
       CHECKPOINT_DISABLE: "1"
-
     steps:
       - uses: actions/checkout@v2
       - name: installing dependencies
-        run: |
-          yarn install
+        run: yarn install
       - run: tools/align-version.sh "-pre.${{ github.run_number }}"
       - run: yarn build
       - run: yarn test
       - run: yarn package
       - run: yarn integration
         env:
           TERRAFORM_CLOUD_TOKEN: ${{ secrets.TERRAFORM_CLOUD_TOKEN }}
+      - name: Upload artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: dist
+          path: dist
+
+  release_npm:
+    name: Release to NPM
+    needs: build_artifact
+    runs-on: ubuntu-latest
+    container:
+      image: hashicorp/jsii-terraform
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This should be downloading the built artifacts and the checkout shouldn't be necessary.

Reply:
good point 👍